instruments/tektronix/tektds5xx.py

```diff
@@ -585,13 +585,14 @@ def get_hardcopy(self):
         """
         self.sendcmd('HARDC:PORT GPI;HARDC:LAY PORT;:HARDC:FORM BMP')
         self.sendcmd('HARDC START')
-        sleep(1)
+        time.sleep(1)
         header = self.query("", size=54)
+        header = header.encode("utf-8")
```

Comment:
Binary encoded data is decoded with "utf-8", thus needs to be encoded again before we can use `struct.unpack` on it.